bobtemplates/plone/view/.mrbob.ini

```diff
@@ -0,0 +1,49 @@
+[questions]
+subtemplate_warning.question = Please commit your changes, before using a sub-template! Continue anyway? [n/y]
+subtemplate_warning.required = True
+subtemplate_warning.default = n
+subtemplate_warning.pre_ask_question = bobtemplates.plone.base:git_clean_state_check
+subtemplate_warning.post_ask_question = mrbob.hooks:validate_choices bobtemplates.plone.base:subtemplate_warning_post_question
+subtemplate_warning.choices = y|n
+subtemplate_warning.choices_delimiter = |
+
+view_python_class.question = Should the view have a Python class?
+view_python_class.help = Do you want Python class for this view?
+view_python_class.required = True
+view_python_class.default = y
+view_python_class.pre_ask_question = bobtemplates.plone.base:check_root_folder
+view_python_class.post_ask_question = mrbob.hooks:validate_choices mrbob.hooks:to_boolean
+view_python_class.choices = y|n
+view_python_class.choices_delimiter = |
+
+view_python_class_name.question = Python class name
+view_python_class_name.help = Shold be something like 'MyView'
+view_python_class_name.required = False
+view_python_class_name.default = MyView
+view_python_class_name.pre_ask_question = bobtemplates.plone.view:check_python_class_answer
+
+view_name.question = View name (part of the URL)
+view_name.help = Should be something like 'my-view' (no special characters!)
+view_name.required = True
+view_name.default = my-view
+view_name.pre_ask_question = bobtemplates.plone.view:get_view_name_from_python_class
+
+
+view_template.question = Should the View have a template file?
+view_template.help = Do you want a template file for this view?
+view_template.required = True
+view_template.default = y
+view_template.post_ask_question = mrbob.hooks:validate_choices mrbob.hooks:to_boolean
+view_template.choices = y|n
+view_template.choices_delimiter = |
+
+view_template_name.question = Template name (without extension)
```

Comment:
the template name, does not have any effect, i chose 
--> Template name (without extension) [my_template_only_view]: my-template-only-view

but the template is named: my_template_only_view.pt

Please test this in a test too.
Also add a skeleton test for the view. The the generate view should have at least one test, which is getting the view and calling a method on it. And check the redendered output too.